webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsService.scala

```diff
@@ -71,64 +72,57 @@ final case class ProjectsServiceLive(bridge: ActorToZioBridge) extends ProjectsS
    *     '''failure''': [[dsp.errors.NotFoundException]] when no project for the given [[ProjectIdentifierADM]] can be found
    */
   def getSingleProjectADMRequest(identifier: ProjectIdentifierADM): Task[ProjectGetResponseADM] =
-    bridge.askAppActor(ProjectGetRequestADM(identifier))
+    responder.getSingleProjectADMRequest(identifier)
 
   /**
    * Creates a project from the given payload.
    *
    * @param payload         the [[ProjectCreatePayloadADM]] from which to create the project
-   * @param requestingUser  the [[UserADM]] making the request
+   * @param user  the [[UserADM]] making the request
    * @return
    *     '''success''': information about the created project as a [[ProjectOperationResponseADM]]
    *
    *     '''failure''': [[dsp.errors.NotFoundException]] when no project for the given [[ProjectIri]]
    *                    can be found, if one was provided with the [[ProjectCreatePayloadADM]]
    *                    [[dsp.errors.ForbiddenException]] when the requesting user is not allowed to perform the operation
    */
-  def createProjectADMRequest(
-    payload: ProjectCreatePayloadADM,
-    requestingUser: UserADM
-  ): Task[ProjectOperationResponseADM] = for {
-    random      <- ZIO.random
-    requestUuid <- random.nextUUID
-    request      = ProjectCreateRequestADM(payload, requestingUser, requestUuid)
-    response    <- bridge.askAppActor[ProjectOperationResponseADM](request)
-  } yield response
+  def createProjectADMRequest(payload: ProjectCreatePayloadADM, user: UserADM): Task[ProjectOperationResponseADM] =
+    ZIO.random.flatMap(_.nextUUID).flatMap(responder.projectCreateRequestADM(payload, user, _))
```

Comment:
I'm not sure I like this change here: the Request ID will soon be added as a ZIO log aspect (DEV-1233 and one of my next tasks), which will mean that it won't fit on one line anymore and then I doubt it will be more readable than before. (I'm ok with merging it like this, but I may revert it when I implement that task)

Reply:
The `apiRequestID` may not be confused with a trace id attached to `Request`. 

Essentially the `apiRequestID` is an identifier for use with the `IriLocker` which describes it as :
>the ID of the API request that needs the lock.

Since the lock is re-entrant, different requests may use the same `apiRequestID`.

Reply:
TLDR: We should not overload the `apiRequestID` original intention with another concern like tracing. I would rather prefer to introduce a `requestTraceId` and rename the `apiRequestID` to something like `lockId`.